src/detectQuickBacks.js

```diff
@@ -24,7 +24,7 @@ export default (subscribe, { threshold = 12000 }) => {
 
   document.addEventListener('click', listener);
 
-  return () => {
+ /* return () => {
```

Comment:
Should it be commented out?

Reply:
Must have left it after testing, removed the comments in the latest commit. Thanks for spotting that :)